examples/testmonitor/results/Results.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections.Generic;
+using NationalInstruments.SystemLink.Clients.TestMonitor;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.TestMonitor
+{
+    /// <summary>
+    /// Example for the SystemLink Test Monitor API that creates a test result
+    /// and associated steps simulating a current and voltage sweep.
+    /// </summary>
+    class Results
+    {
+        /// <summary>
+        /// Creates a <see cref="StepData"/> object and
+        /// populates it to match the TestStand data model.
+        /// </summary>
+        /// <param name="name">The test step's name.</param>
+        /// <param name="stepType">The test step's type.</param>
+        /// <param name="current">The value of the measured electrical current.</param>
+        /// <param name="voltage">The value of the measured electrical voltage.</param>
+        /// <param name="power">The value of calculated electrical power.</param>
+        /// <param name="lowLimit">The value of the low test limit.</param>
+        /// <param name="highLimit">The value of the high test limit.</param>
+        /// <returns>The <see cref="StepData"/> used to create a test step.</returns>
+        private static StepData GenerateStepData(string name, string stepType, double current, double voltage, double power, double lowLimit, double highLimit)
+        {
+            Random random = new Random();
+            var status = new Status(StatusType.Running);
+            var inputs = new List<NamedValue>();
+
+            var outputs = new List<NamedValue>();
+
+            var parameters = new List<Dictionary<string, string>>();
+            if (stepType.Equals("NumericLimit"))
+            {
+                // Add parameters to match the TestStand data model
+                var parameter = new Dictionary<string, string>();
+                parameter.Add("name", "Power Test");
+                parameter.Add("status", "status");
```

Comment:
```suggestion
                parameter.Add("status", status.StatusType.ToString());
```

We should also probably set this below line 62 when we know the status of the measurement. Or, we could leave it out entirely and just let the step status encapsulate all the information.